app/services/report_service.rb

```diff
@@ -50,32 +50,8 @@ def notify_staff!
     end
   end
 
-  def forward_to_origin!
-    return unless forward_to_origin?
-
-    # Send report to the server where the account originates from
-    ActivityPub::DeliveryWorker.perform_async(payload, some_local_account.id, @target_account.inbox_url)
-  end
-
-  def forward_to_replied_to!
-    # Send report to servers to which the account was replying to, so they also have a chance to act
-    inbox_urls = Account.remote.where(domain: forward_to_domains).where(id: Status.where(id: reported_status_ids).where.not(in_reply_to_account_id: nil).select(:in_reply_to_account_id)).inboxes - [@target_account.inbox_url, @target_account.shared_inbox_url]
-
-    inbox_urls.each do |inbox_url|
-      ActivityPub::DeliveryWorker.perform_async(payload, some_local_account.id, inbox_url)
-    end
-  end
-
   def forward?
-    !@target_account.local? && ActiveModel::Type::Boolean.new.cast(@options[:forward])
-  end
-
-  def forward_to_origin?
-    forward? && forward_to_domains.include?(@target_account.domain)
-  end
-
-  def forward_to_domains
-    @forward_to_domains ||= (@options[:forward_to_domains] || [@target_account.domain]).filter_map { |domain| TagManager.instance.normalize_domain(domain&.strip) }.uniq
```

Comment:
If I'm understanding what this code did correctly, then if the forward_to_domains option wasn't specified, then it'd always forward to the target_account's domain. We may want to reassess that, and instead rely on manual forwarding — I don't know if a missing API parameter should be seen as "yes, let's forward this to a remote instance"